dkist/io/asdf/converters/dataset.py

```diff
@@ -1,19 +1,18 @@
-from dkist.dataset import Dataset
+from asdf.extension import Converter
 
-from ..types import DKISTType
 
-__all__ = ["DatasetType"]
+class DatasetConverter(Converter):
+    tags = [
+        "tag:dkist.nso.edu:dkist/dataset-0.1.0",
+        "tag:dkist.nso.edu:dkist/dataset-0.2.0",
+        "tag:dkist.nso.edu:dkist/dataset-0.3.0",
+        "asdf://dkist.nso.edu/dataset-1.0.0",
```

Comment:
Elsewhere we've been putting tags into a /tags/ namespace (like with /schemas/).  This works just as well, but if you're interested in uniformity...